File: 1-Beginner/1014/bee1014.c
// 1014 - Consumo
#include <stdio.h>

int main(void) {

    int distancia;
    double combustivel;
    scanf("%d%lf", &distancia, &combustivel);

    printf("%.3lf km/l\n", distancia/combustivel);

    return 0;
}

File: 1-Beginner/1014/bee1014.java
// 1014 - Consumo
import java.io.IOException;
import java.util.Scanner;

class bee1014 {

    public static void main(String[] args) throws IOException{
        Scanner scan = new Scanner(System.in);

        int distancia = scan.nextInt();
        double combustivel = scan.nextDouble();

        System.out.printf("%.3f km/l\n", distancia/combustivel);

        scan.close();
    }
}